models/ios/ntp_global/commands.txt

```diff
@@ -0,0 +1,180 @@
+THIS FILE WONT BE PUSHED TO THE MAIN REPO
```

Comment:
Looks like it was pushed to the main repo? ;)

Reply:
it somehow made it here :D i'll remove it post the reviews